modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -89,6 +98,12 @@
         },
       },
     },
+    'cloud'                    => {
+      'aws' => {
+        'access_key' => $aws_access_key,
+        'secret_key' => $aws_secret_key,
+      },
+    },
```

Comment:
The permissions on the file this ends up in are 644, so I think anyone on the box will be able to see it. Not sure how big of a deal that is though.